db/external_sst_file_test.cc

```diff
@@ -1249,8 +1249,9 @@ TEST_P(ExternalSSTFileTest, PickedLevel) {
 
   // This file overlaps with file 0 (L3), file 1 (L2) and the
   // output of compaction going to L1
-  ASSERT_OK(GenerateAndAddExternalFile(options, {4, 7}, -1, false, false, true,
-                                       false, false, &true_data));
+  ASSERT_OK(GenerateAndAddExternalFile(options, {4, 7}, -1,
```

Comment:
[Note] Similar to above, ingest file of overlapped ranges with ongoing compaction needs `allow_global_seqno=true` just like how it's needed for overlapping ranges with existing files.